Tests/ChainsafeRPCTests.cs

```diff
@@ -12,52 +11,56 @@ namespace Tests
+            
+            // Deploy a contract with https://remix.ethereum.org/ and your Ganache instance.
+            // Make sure to select Ganache Provider as the Environment. Once deployed, replace the _contractAddress value with your contract address
+            _contractAddress = "0x2061c2B28F007DD0eD5064A61d352521CC1d2827";
```

Comment:
Big change here, because we are not using Infura anymore, to be able to run 3 of the tests, you need to deploy a contract using the Ganache instance